examples/automm/Kaggle-Pawpularity/README.md

```diff
@@ -0,0 +1,103 @@
+﻿Take [Petfinder Pawpularity competition](https://www.kaggle.com/competitions/petfinder-pawpularity-score/overview) as an example showing how to use AutoMMPredictor to complete a competition.
+
+## 1. Kaggle Kernel-only Competition with AutoGluon
+
+In a kaggle competition, especially a code competition, users cannot obtain AutoGluon resources through the network. 
+To solve the problem, there are two key points:
+
+ - Get AutoGluon and its dependent libraries.
+ - Save the downloaded resources when saving your model.
+
+### 1.1 Get AluoGluon in Kaggle
+
+You can upload the installed AutoGluon to kaggle as a dataset and import the dataset into codes. In this way, autogluon is introduced without network support in submission. 
+Through the code follow, you can import AutoGluon in your work.
+
+    import sys
+    sys.path.append("../input/autogluon/")
+  
+ It should be noted that AutoGluon itself needs some dependencies which are not supported in the kaggle environment. They should be introduced the same way as Auotgluon. 
+Currently, these libraries need to be imported manually:
+
+ - nptyping
+ - typish
+ - timm
+ - omegaconf
+ - antlr4
+
+### 1.2 Save Standalone Model
+In AutoMMPredictor, some pretrained models will be downloaded during training. These models also need to be saved for use in predicting after submission. You can open standalone in model saving to save the downloaded models.
+
+    predictor.save(path=save_standalone_path,standalone=True)
+
+## 2. AutoMMPredictor for Training
```

Comment:
I suggest to move this section above. We first teach the user about how to train the model and then teach them how to submit it in Kaggle.